src/components/UI/Display/Apps/AppList/CatalogLabel.tsx

```diff
@@ -63,14 +64,37 @@ ErrorIcon.propTypes = {
 };
 
 const CatalogLabel: React.FC<ICatalogLabelProps> = (props) => {
+  const text = (
+    <span>
+      {props.catalogName}
+      {props.isManaged && <CatalogType>MANAGED</CatalogType>}
+      {props.catalogName === 'Giant Swarm Catalog' && (
+        <CatalogType>MANAGED</CatalogType>
+      )}
+    </span>
+  );
+
   return (
     <Wrapper {...props} hasError={Boolean(props.error)}>
       <IconArea>
         <Icon src={props.iconUrl} />
       </IconArea>
       <Text>
-        {props.catalogName}
-        {props.isManaged && <CatalogType>MANAGED</CatalogType>}&nbsp;
+        {props.description && (
+          <OverlayTrigger
+            placement='top'
+            overlay={
+              <Tooltip id={`app-catalog-description-${props.catalogName}`}>
+                {props.description}
+              </Tooltip>
+            }
+          >
+            {text}
+          </OverlayTrigger>
+        )}
+
+        {(!props.description || props.description === '') && text}
```

Comment:
Doesn't `!props.description` also equal to `true` when it's an empty string?

Reply:
Yes! but you'd think I'd have learned that by now right. :P